cmd/crl-checker/main.go

```diff
@@ -61,6 +64,7 @@ func main() {
 	issuerFile := flag.String("issuer", "", "path to an issuer certificate on disk")
 	ageLimitStr := flag.String("ageLimit", "168h", "maximum allowable age of a CRL shard")
 	emitRevoked := flag.Bool("emitRevoked", false, "emit revoked serial numbers on stdout, one per line, hex-encoded")
+	save := flag.Bool("save", false, "save CRLs to files named after the last component of their URL")
```

Comment:
It looks like the file is named after the whole url (host and path), not just the last component of the url?